src/cup-muffin.md

```diff
@@ -0,0 +1,39 @@
+# Muffin
+
+Quick and easy recipe of chocolate muffin in a cup.
+
+- ⏲️ Prep time: 2 min
+- 🍳 Cook time: 90 sec
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- 3 tablespoons of flour
+- 1 teaspoon of instant coffee
+- 1/2 tablespoon of cocoa-powder
+- 3 tablespoons of sugar
+- 1/4 teaspoon of baking powder
+- 4 tablespoons of milk
+- 1 egg
+- 2 tablespoons of vegetable oil
+- 1/2 teaspoon of vanilla extract
+
+## Directions
+
+1. Mix the dry ingredients (flour, coffee, cocoa powder, sugar and baking powder) in a medium bowl.
+2. Add milk, egg, oil and vanilla extract. Mix well until homogeneous mass is formed.
+3. Pour the mass into little greased cup and put in the microwave on the highest mode for about 90 seconds.
+4. You can serve it with a ball of ice cream.
+
+## Recommendations
+
+1. Use a small cup.
+2. Better with coconut oil.
+3. Fill a third of the cup with the mass.
+
+## Contribution
+
+- Leonid Semenov - [github](https://github.com/LnSxx)
+- btc: `14RGHz3iepA3xTNYc64MP3EPq7rDjKVSWQ`
+
+;tags: cake dessert quick sweet
```

Comment:
4 tags are a lot, I think 3 will suffice.

Reply:
@Knarkzel The automated PR check script currently allows anything between 2 and 5 tags. Doesn't really matter to me what the number of allowed tags are, but whatever we decide is allowed should probably be documented and consistent. Maybe the script should be updated soon